README.md

```diff
@@ -1 +1,57 @@
-# example-scripts
\ No newline at end of file
+# Example automation scripts using JupiterOne platform
+
+This repo provides a few simple examples on how to maintain security/compliance
+as code and to automate SecOps using the JupiterOne platform.
+
+The examples are in either `bash` or `javascript`.
+
+These scripts are provided as is. For questions, please post in the `#dev`
+channel in `jupiterone-community` Slack workspace.
```

Comment:
Can this be made a link to the Slack community? How about:

For questions, please post in the [`jupiterone-community #dev`](https://jupiterone-community.slack.com/messages/CJMV4SFV5) Slack workspace.